app/javascript/actions/greetingActions.js

```diff
@@ -0,0 +1,57 @@
+// import axios from 'axios';
+
+// export const setGreeting = (greeting) => ({
+//   type: 'SET_GREETING',
+//   payload: greeting,
+// });
+
+// export const fetchGreeting = () => {
+//   return async (dispatch) => {
+//     try {
+//       const response = await axios.get('http://localhost:3000/greeting');
+//       console.log(response.data.greeting)
+//       dispatch(setGreeting(response.data.greeting));
+//     } catch (error) {
+//       // Handle error
+//       console.log(error);
+//     }
+//   };
+// };
+
+
+
+
+// import axios from 'axios';
+
+// export const setGreeting = (greeting) => ({
+//   type: 'SET_GREETING',
+//   payload: greeting,
```

Comment:
Nice job so far👍🏼. Just as suggested previously, you still haven't taken the commented block of code from your code base. Kindly do well to remove them from your codebase as they make it look messy and give it a less professional look.

Reply:
Good observation but I did so on my vscode and it did not show in my repo that is I called you to justify...